src/EA.Weee.RequestHandlers/Admin/GetAllAatfsForComplianceYearHandler.cs

```diff
@@ -35,7 +36,9 @@ public async Task<List<OrganisationSchemeData>> HandleAsync(GetAllAatfsForCompli
                 throw new ArgumentException($"Aatfs not found for a compliance year {message.ComplianceYear}");
             }
 
-            return mapper.Map<List<Domain.AatfReturn.Aatf>, List<OrganisationSchemeData>>(listAatfs);
+            var result = mapper.Map<List<Domain.AatfReturn.Aatf>, List<OrganisationSchemeData>>(listAatfs);
```

Comment:
Possible to add a unit test to check the ordering... or update the existing one.

Reply:
I have unit tests for the handler, they are in PR #51587